### DISCUSSION TITLE
next.config.js

### DISCUSSION THREAD

**Discussion by lotfy1654** [vercel/next.js#36783] [Help]
Error: Invalid src prop [(https://via.placeholder.com/150/92c952)](http://localhost:3000/(https://via.placeholder.com/150/92c952)) on `next/image`, hostname "via.placeholder.com" is not configured under images in your `next.config.js`
See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host

i add it but don't work
module.exports = {
    images: {   
      domains: ['https://via.placeholder.com/']
     },
}

**Comment by vadymshymko** [comment]
Hi. Try add only domain, no protocol:
```
module.exports = {
  images: {
    domains: ['via.placeholder.com']
  },
}
```

**Comment by juancruzfl** [comment]
I had a similar issue but solved it by doing something along the lines of this: 
```sh
const nextConfig = {
  images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: "via.placeholder.com",
        port: '',
        pathname: '/**',
      },
      
    ],
  },
}
```
Adjust the path property to restrict the allowed paths you want to load images with the hostname URL.
